Improving embed output in the history channel

sabihoshi opened the pull request:
This always places the guild locale as the first field, making it more readable and will make you be able to follow the conversation.

![preview](https://awau.moe/6KgnDXK.png)

sabihoshi committed:
Improve embed output in history channel

The guild locale is now always the first field to appear, as well as the icon of the user will appear to make it more readable.

Cisien commented:
Unfortunately, the field value is limited to 1024 characters, where as a discord message can be up to 2000 characters in length, forcing us to truncate up to half of the message, which would diminish the value of this channel.

The way the embed is currently is like that because of these restrictions. An improvement in the current form that I had thought about would be to put the target language version of the message as the standard message, and embed the english version, that way it's at least consistent.

sabihoshi commented:
Descriptions could have 2048 as a limit, so the important part (the guild language) will be the description, meanwhile the other could be split up. Or the whole thing will have to be in fields, if you want to fully preserve history, as text could become bigger than 2000 characters when translated.

Cisien commented:
Yeah, I would like it fully preserved. 

1. anyone searching either in english or a translated language would find a thread that might help them
2. moderation. Having a message log is important after the channel expires.

sabihoshi committed:
Make sure to not exceed character limits in embeds

Translations still look better side to side, unless there is a code block. In case any one of them exceeds 1024 limit, it will be split into chunks and all of them will be inlined.

sabihoshi commented:
Fields will still be inlined when it's short enough, as it's easier to read them side by side, at least it makes you be able to follow one language by sticking your eyes in just one side. As before, the primary locale of the server is at the left.
![image](https://user-images.githubusercontent.com/25006819/84598942-f2385580-aea0-11ea-8e6c-5e0a95de8e82.png)

If any of the chunks is greater than 1024, then it will be have to split. This also includes code blocks, as code blocks being inlined will make it too thin to be actually be readable.
![image](https://user-images.githubusercontent.com/25006819/84598974-257ae480-aea1-11ea-979b-06b30eb1573c.png)
